groups/advanced-search/locals.tf

```diff
@@ -0,0 +1,14 @@
+locals {
+  secrets = data.vault_generic_secret.secrets.data
+
+  accessible_from_subnet_patterns = jsondecode(local.secrets.accessible_from_subnet_patterns)
+  availability_zone_count = length(data.aws_subnet_ids.placement.ids)
+  internal_cidrs = values(data.terraform_remote_state.networking.outputs.internal_cidrs)
```

Comment:
See above—this could be replaced with a Vault data resource rather than remote state.